tlslite/recordlayer.py

```diff
@@ -518,8 +520,10 @@ def sendRecord(self, msg):
 
         # TLS 1.3 hides the content type of messages
         if self._is_tls13_plus() and self._writeState.encContext:
-            # TODO - add support for padding
             data += bytearray([contentType])
+            if self.padding_cb:
+                # add number of zero bytes specified by padding_cb()
+                data += bytearray(self.padding_cb(len(data), contentType))
```

Comment:
callback should get the maximum size of padding it can add as one of the parameters